barretenberg/cpp/src/barretenberg/vm/avm/trace/helper.cpp

```diff
@@ -108,4 +263,16 @@ std::string to_hex(bb::avm_trace::AvmMemoryTag tag)
     return to_hex(static_cast<uint8_t>(tag));
 }
 
+using field_ct = stdlib::field_t<UltraCircuitBuilder>;
+
+template VmPublicInputs<FF> convert_public_inputs(std::vector<FF> const& public_inputs_vec);
+template VmPublicInputs<field_ct> convert_public_inputs(std::vector<field_ct> const& public_inputs_vec);
+
+template std::vector<std::vector<FF>> copy_public_inputs_columns(VmPublicInputs<FF> const& public_inputs,
+                                                                 std::vector<FF> const& calldata,
+                                                                 std::vector<FF> const& returndata);
+template std::vector<std::vector<field_ct>> copy_public_inputs_columns(VmPublicInputs<field_ct> const& public_inputs,
+                                                                       std::vector<field_ct> const& calldata,
+                                                                       std::vector<field_ct> const& returndata);
```

Comment:
I think for this helper it might be simpler to move the impl to the hpp and avoid having to do any explicit instantiation (+ avoid the includes).

Reply:
Done.